test/Riok.Mapperly.Tests/Mapping/ObjectPropertyNullableTest.cs

```diff
@@ -588,6 +588,45 @@ public partial B Map(A a)
             );
     }
 
+    [Fact]
+    public void NullableNestedMembersShouldInitializeWithNoNullAssignment()
```

Comment:
Could you also create a test case for this slightly different one: https://github.com/riok/mapperly/issues/1661 (initialization on another level)

Reply:
I added such a test 😊